src/pubsub/ua_pubsub.h

```diff
@@ -200,6 +200,9 @@ typedef struct UA_DataSetReader {
 /* Process Network Message using DataSetReader */
 void UA_Server_DataSetReader_process(UA_Server *server, UA_DataSetReader *dataSetReader, UA_DataSetMessage* dataSetMsg);
 
+/* Process Network Message in RT configuration using DataSetReader */
+void UA_Server_DataSetReader_processRTMessage(UA_Server *server, UA_DataSetReader *dataSetReader);
```

Comment:
Please use `UA_Server_DataSetReader_process` also for the RT path.

You can split this the RT path out **inside of** UA_Server_DataSetReader_process.
But everything should go through the same entry point for processing.

Reply:
Done